pkg/email/grb_review_discussion_individual_tagged.go

```diff
@@ -78,12 +84,10 @@ func (sie systemIntakeEmails) SendGRBReviewDiscussionIndividualTaggedEmail(ctx c
 		return err
 	}
 
-	allRecipients := input.Recipients
-
 	return sie.client.sender.Send(
 		ctx,
 		NewEmail().
-			WithToAddresses(allRecipients). // TODO: NJD cc and/or bcc?
+			WithToAddresses(input.Recipients). // TODO: NJD cc and/or bcc?
```

Comment:
Same feedback here as the other method (making it take a single recipient)

I also don't think we technically will call this method with `Role: ""` since the only people we can individually tag are the GRB reviewers, but it doesn't necessarily hurt to implement what you did here if we do support that eventually.

Reply:
updated this and the above to only take one recipient - good call! thanks